test_rigs/parsing-tests/unit-tests.ref

```diff
@@ -11722,10 +11736,20 @@
             "      (ss_host",
             "        HOST:'host'  <== mode:DEFAULT_MODE",
             "        IP_ADDRESS:'10.1.2.3'  <== mode:DEFAULT_MODE",
-            "        (ss_host_use_vrf",
-            "          USE_VRF:'use-vrf'  <== mode:DEFAULT_MODE",
+            "        (ss_host_null",
             "          (variable",
-            "            MANAGEMENT:'management'  <== mode:DEFAULT_MODE)",
+            "            VARIABLE:'xyx'  <== mode:DEFAULT_MODE)",
```

Comment:
This is a breaking change. See comment above about `variable` in `ss_host_null`.